Sources/ProjectDescription/Dependencies/Pod.swift

```diff
@@ -0,0 +1,206 @@
+import Foundation
+
+/// A Pod dependency.
+///
+/// A Pod dependency can be either:
+///     - remote: A Git URL to the source of the package,
+///     and a requirement for the version of the package.
+///     - local: A relative path to the package.
+public enum Pod: Equatable, Codable {
+    case remote(url: String, requirement: Requirement)
+    case local(path: Path)
+
+    private enum Kind: String, Codable {
+        case remote
+        case local
+    }
+
+    private enum CodingKeys: String, CodingKey {
+        case kind
+        case url
+        case productName
+        case requirement
+        case path
+    }
+
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        let kind = try container.decode(Kind.self, forKey: .kind)
+        switch kind {
+        case .remote:
+            let url = try container.decode(String.self, forKey: .url)
+            let requirement = try container.decode(Requirement.self, forKey: .requirement)
+            self = .remote(url: url, requirement: requirement)
+        case .local:
+            let path = try container.decode(Path.self, forKey: .path)
+            self = .local(path: path)
+        }
+    }
+
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+        switch self {
+        case let .remote(url, requirement):
+            try container.encode(Kind.remote, forKey: .kind)
+            try container.encode(url, forKey: .url)
+            try container.encode(requirement, forKey: .requirement)
+        case let .local(path):
+            try container.encode(Kind.local, forKey: .kind)
+            try container.encode(path, forKey: .path)
+        }
+    }
+}
+
+extension Pod {
+    public enum Requirement: Codable, Equatable {
```

Comment:
If we end up having the same requirement as the Packages' I'll extract this into `DependencyRequirement` and share it across package and pod dependencies.

Reply:
> and share it across package and pod dependencies

👍 

The extracting will be needed in the `TuistGraph` target too.